scipy/special/__init__.py

```diff
@@ -640,7 +641,7 @@
 from ._ufuncs import *
 
 from .basic import *
-from ._logsumexp import logsumexp
+from ._logsumexp import logsumexp, softmax
```

Comment:
Re "_logsumexp is a bad filename": it doesn't bother me too much, but it's okay to rename that file. Maybe `_logspace_funcs.py`?